Package.swift

```diff
@@ -10,7 +10,7 @@ let package = Package(
     .package(url: "https://github.com/swift-server/async-http-client.git", from: "1.1.1"),
     .package(url: "https://github.com/apple/swift-argument-parser", .upToNextMinor(from: "0.2.0")),
     .package(url: "https://github.com/apple/swift-tools-support-core.git", .upToNextMinor(from: "0.1.3")),
```

Comment:
<!--
  1 failure:  Line should be 10...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: Package.swift;
  Line: 12;
-->
- :no_entry_sign: Line should be 100 characters or less: currently 106 characters (`line_length`)